lib/zendesk/ticket.ex

```diff
@@ -65,6 +68,25 @@ defmodule Zendesk.Ticket do
     %Operation{path: "tickets.json", parser: &parse_list/1} |> Operation.with_page_size(1)
   end
 
+  @doc """
+  Returns the full ticket object as it would be after applying the macro to the ticket.
+
+  It doesn't actually change the ticket.
+  """
+  @spec after_changes(non_neg_integer(), non_neg_integer()) :: Operation.t()
+  def after_changes(%Ticket{id: ticket_id}, %Macro{id: macro_id}) do
+    %Operation{path: "tickets/#{ticket_id}/macros/#{macro_id}/apply.json", parser: &parse/1}
+  end
+
```

Comment:
```suggestion
def after_changes(ticket_id, macro_id) do
  %Operation{path: "tickets/#{ticket_id}/macros/#{macro_id}/apply.json", parser: &parse/1}
end
```

Reply:
I think in most cases there will be a ticket / macro lookup before calling this function (which is why I updated to accept a ticket/macro instead).  I was thinking of just holding off on accepting ints until/unless usage/ergonomics lead to wanting it. Open to adding though 🤷‍♂️

Reply:
Agreed. Huey Lewis might disagree, but I think one needs a Ticket to ride this train. And a Macro.